ci/local-run.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+function check_var() {
+  [[ ${!1-} ]] && return 0 || true
+  [[ -f .env ]] || { echo "Var $1 missing from environment and no .env file found; see .env.sample file"; return 1; }
+  grep -q "^$1=" .env || { echo "Var $1 missing both from environment and .env file"; return 1; }
+}
+
+image_name=$1
+
+if [[ $OSTYPE == 'darwin'* ]]; then  # group id 20 is usually taken in MacOS
+  group_ID="500"
+else
+  group_ID="$(id -g)"
+fi
+
+cd "$(git rev-parse --show-toplevel)"/ci/$image_name/
+if [[ "$image_name" == "tox" ]]; then
+  cp .env-sample .env  # and optionally edit if needed to change git/python version
```

Comment:
Hmmm should this `cp` be done by a script? This way whatever the user kept in their .env file will be overwritten with the sample :/